cmd/kube-apiserver/app/server.go

```diff
@@ -568,7 +570,9 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 	}
 
 	if s.Etcd.EncryptionProviderConfigFilepath != "" {
-		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(s.Etcd.EncryptionProviderConfigFilepath)
+		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(
```

Comment:
What exactly are you referring to here, when you say "library"? Can you elaborate a little bit more on how you'd want these few lines to look?